### ISSUE TITLE
Accessibility issue with CvOverflowMenu component

### ISSUE THREAD

**Issue by caroluchoa** [carbon-design-system/carbon-components-vue#1350]
## Detailed description
This issue is related to CvOverflowMenu. Using IBM Accessibility Checker on the CvOverflowMenu results on the following violation: "Form control element <button> has no associated label. Violation"
I have uploaded a print of accessibility checker scan showing the violation and the code.

## Additional information

- Screenshots or code

<img width="685" alt="image" src="https://user-images.githubusercontent.com/43659990/180068842-a379f511-855c-4786-8a0b-abb2dcbd47a2.png">

Piece of code that caused the violation:

```html
<cv-overflow-menu id="header-overflow" :flip-menu="true" aria-label="menu">
       <cv-overflow-menu-item @click="applicationHistory" aria-label="History" </cv-overflow-menu-item>
       <cv-overflow-menu-item @click="editApplication" aria-label="Edit application">
           Edit application
       </cv-overflow-menu-item>
       <cv-overflow-menu-item @click="deleteApplication" aria-label="Delete application">
           Delete application
       </cv-overflow-menu-item>
</cv-overflow-menu>
```
